dbms/src/Interpreters/ClusterProxy/TruncateStreamFactory.h

```diff
@@ -0,0 +1,31 @@
+#pragma once
+
+#include <Interpreters/ClusterProxy/IStreamFactory.h>
+
+namespace DB
+{
+
+namespace ClusterProxy
+{
+
+class TruncateStreamFactory final : public IStreamFactory
```

Comment:
Missing comment.
I think, we don't need special support for TRUNCATE of Distributed tables.

Reply:
- Maybe I can add `push_distributed_operation` setting to decide whether to push truncate?
- Currently distributed tables only push read and write operations, which is always not very friendly, I'm not sure if there is a plan to support it ?